cfme/tests/automate/test_common_methods.py

```diff
@@ -250,3 +253,73 @@ def test_service_retirement_from_automate_method(request, generic_catalog_item,
         description=f"Service Retire for: {generic_catalog_item.name}")
     wait_for(lambda: retire_request.request_state == "finished", fail_func=retire_request.reload,
              timeout=180, delay=10)
+
+
+@pytest.fixture
+def set_roottenant_quota(request, appliance):
```

Comment:
**Suggestion:**
Can you rename it to something like: `set_root_tenant_quota` .?